Automattic/wp-calypso
# Site Management Panel: Update primary styling on buttons

Related to https://github.com/Automattic/dotcom-forge/issues/8783

## Proposed Changes

* This PR removes the "primary" button styling from the buttons in the Site Management flyout panels.



Before | After
--|--
<img width="1728" alt="Screenshot 2024-09-04 at 10 15 23 AM" src="https://github.com/user-attachments/assets/61a66e51-3498-4eb9-9585-fa0656d0d0a1">  | <img width="1726" alt="Screenshot 2024-09-04 at 10 15 05 AM" src="https://github.com/user-attachments/assets/c1d9fc59-7926-4820-adcf-bcfe708e09a6">


Before | After
--|--
<img width="1727" alt="Screenshot 2024-09-04 at 10 15 34 AM" src="https://github.com/user-attachments/assets/3916ddca-0979-49d9-8303-af640909dc76"> | <img width="1728" alt="Screenshot 2024-09-04 at 10 15 44 AM" src="https://github.com/user-attachments/assets/18c0d70d-0b6e-47f4-aadf-cb51b19d573b">

Before | After
--|--
<img width="1728" alt="Screenshot 2024-09-04 at 10 16 04 AM" src="https://github.com/user-attachments/assets/bfa0e156-226a-46d9-be4c-673ce1ac7c36"> | <img width="1728" alt="Screenshot 2024-09-04 at 10 15 55 AM" src="https://github.com/user-attachments/assets/781eeb97-fd9e-4e5a-9b5c-99337650852d">

Before | After
--|--
<img width="1728" alt="Screenshot 2024-09-04 at 10 37 39 AM" src="https://github.com/user-attachments/assets/28ad0ed7-2786-4d0a-93ea-e8f150c5c05f">  | <img width="1728" alt="Screenshot 2024-09-04 at 10 37 28 AM" src="https://github.com/user-attachments/assets/7601091b-99cd-4db6-b4ed-3e3f409ea194">





Before | After
--|--
<img width="1728" alt="Screenshot 2024-09-04 at 10 16 19 AM" src="https://github.com/user-attachments/assets/03efed25-4cd9-4c93-b29a-9c88fbb3ddd8"> | <img width="1728" alt="Screenshot 2024-09-04 at 10 16 29 AM" src="https://github.com/user-attachments/assets/cf1d7bae-eeca-4fa5-80f1-fdf0d0fb577f">

Before | After
--|--
<img width="1728" alt="Screenshot 2024-09-04 at 10 45 40 AM" src="https://github.com/user-attachments/assets/18b6b876-0156-40ac-be6f-37093893aeb1">  | <img width="1728" alt="Screenshot 2024-09-04 at 10 46 07 AM" src="https://github.com/user-attachments/assets/05544db5-fcb3-4a4d-b2a7-ae700945a9cb">



## Why are these changes being made?


* Decrease competition for the user's attention with too many call-to-action styled buttons.

## Testing Instructions



* Use Calypso Live
* Go to /sites and check all of Site Management tabs for "primary" buttons. They should be "regular" buttons, like in the screenshots above.

## Pre-merge Checklist
